staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -104,6 +122,15 @@ func TestParserV1alpha1(t *testing.T) {
 	}
 }
 
+func TestParsePolicyWithNoVersionOrKind(t *testing.T) {
+	f, err := writePolicy(t, policyWithNoVersionOrKind)
+	require.NoError(t, err)
+	defer os.Remove(f)
+
+	_, err = LoadPolicyFromFile(f)
+	require.Error(t, err)
```

Comment:
I'd like to replace this line with:
```
assert.Containsf(t,err.Error(), "policy file contained invalid apiVersion and kind field combination", "a policy without apiVersion or Kind should not be accepted.")
```

So that we are sure this err is not caused by other reason